app/safecast/actions/bgeigie_imports/kml.rb

```diff
@@ -17,7 +17,7 @@ def execute(context, filename = nil)
         'app/views/bgeigie_imports/bgeigie_logs.kml.erb'
 
       def send_opts(filename)
-        opts = { type: Mime::KML.to_s }
+        opts = { type: Mime::Type.lookup('kml').to_s }
```

Comment:
I think that we need MIME type here.

```suggestion
        opts = { type: Mime[:kml].to_s }
```

```
[1] pry(main)> Mime::Type.lookup('kml').to_s
=> "kml"
[2] pry(main)> Mime[:kml].to_s
=> "application/vnd.google-earth.kml+xml"
```